### PR TITLE
Add test for Decrement, Increment, View, Sort, Help, Exit and Reset

### PR DESCRIPTION
**Pull request by yixiann**
Did some refactoring of referencing static fields. 

Below are some notes regarding bugs found. All are fixed.

Sorting:

- Wrong ID was being referenced.
- Sorting by many fields require argmultimap to be refactored to deduce relative position of each prefix. Sorting was reverted back to only being able to sort by a single field. Instead of using isValuePresent(Prefix prefix), preamble was used. Reason being we need an input with the prefix to be detected.
- Testing sorting using the new implementation proved difficult. Thus I did all the testing manually.

Increment / Decrement:

- Performing command will remove all tags from the item. 

Exit:

- Comment provided to user on exit says AB3 instead of FoodRem.

Notes: 

- Many lines are attributed to creating a file to test sorting effectively (1000+). 
- Also to not directly import static strings.

### CODE REVIEW

File: src/main/java/seedu/foodrem/commons/enums/CommandType.java [hunk 1]
```diff
@@ -156,20 +156,20 @@ public String getUsage() {
         }
     },
     SORT_COMMAND("sort") {
-        // TODO: Ensure remarks is added
         @Override
         public String getUsage() {
             return getCommandWord() + ": Sorts all items in FoodRem according to the specified criteria.\n"
                     + "Available criteria includes sorting by name, quantity, unit, bought date, expiry date "
```
Comment (RichDom2185) [inline src/main/java/seedu/foodrem/commons/enums/CommandType.java:162]:
```suggestion
                    + "Available criteria includes sorting by name, quantity, unit, bought date, expiry date, "
```

Comment (RichDom2185) [reply src/main/java/seedu/foodrem/commons/enums/CommandType.java:162]: Please update the test as well.

**Review by RichDom2185 — DISMISSED** [review]
LGTM, just a few small fixes. Thank you so much for the hard work! Let me know when it's no longer a draft 😄

**Review by bryanljx — APPROVED** [review]
Looks good

**Pull request merged**